Assets/Scripts/Layers/LayerTypes/RootLayer.cs

```diff
@@ -15,6 +13,13 @@ public class RootLayer : LayerData
 
         public RootLayer(string name) : base(name)
         {
+           
+        }
+
+        public void Initialize()
+        {
+            ReconstructParentsRecursive();
+            ProjectData.Current.OnDataChanged.AddListener(data => UpdateLayerTreeOrder(0));
```

Comment:
Gaat dit geen probleem geven als ik meerdere projecten achter elkaar zou inladen? Want IIRC wordt ProjectData niet gedestroyed, alleen maar vervangen wat erin zit. En dat zou betekenen dat elke RootLayer in geheugen blijft en van elk de UpdateLayerTree order zou worden aangeroepen?

Reply:
goed punt ga ik fixen